public/js/index.js

```diff
@@ -3122,3 +3122,19 @@ function onOAuthLoginClick() {
         oauthNav.show();
     },5000);
 }
+
+(function addLabelFilterClickEvents() {
```

Comment:
This should be moved to appropiate Init function

Reply:
Moved